utils/build.ps1

```diff
@@ -1882,7 +1871,7 @@ function Build-XCTest([Platform]$Platform, $Arch, [switch]$Test = $false) {
       $env:Path = "$XCTestBinaryCache;$FoundationBinaryCache\bin;$DispatchBinaryCache;$(Get-TargetProjectBinaryCache $Arch Runtime)\bin;$env:Path;$UnixToolsBinDir"
     } else {
       $TestingDefines = @{ ENABLE_TESTING = "NO" }
-      $Targets = @("default")
+      $Targets = @("install")
```

Comment:
Could this `$Target` to be empty instead of having "install" like other places?

Reply:
Yes, this side can be empty, good idea.